src/cascadia/TerminalApp/SuggestionsControl.xaml

```diff
@@ -30,7 +30,6 @@
                               MinHeight="0"
                               Padding="16,0,12,0"
                               HorizontalContentAlignment="Stretch"
-                              AutomationProperties.AcceleratorKey="{x:Bind Item.KeyChordText, Mode=OneWay}"

```

Comment:
This looks like it's completely gone now. But, it looks like we weren't ever showing the associated key chord in the sxn ui.  I don't use sxn ui that much tho. Is that the case?

Reply:
Yep, the `KeyChordText` was never there in the first place!